### PR TITLE
Rotate logo

### PR DESCRIPTION
**Pull request by fortanza**
Create component Rotate logo

### CODE REVIEW

**Review by Sonya092 — APPROVED** [review]

**Comment by Sonya092** [comment]
Good to me

**Review by Sonya092 — APPROVED** [review]
Good too

**Review by Sonya092 — APPROVED** [review]

**Review by Sonya092 — APPROVED** [review]

**Review by Sonya092 — APPROVED** [review]

**Review by Sonya092 — APPROVED** [review]

**Review by Sonya092 — APPROVED** [review]

**Review by Sonya092 — APPROVED** [review]

**Review by Sonya092 — APPROVED** [review]

**Review by Sonya092 — APPROVED** [review]

**Review by Sonya092 — APPROVED** [review]

**Review by Sonya092 — APPROVED** [review]

**Review by Sonya092 — APPROVED** [review]

**Review by Sonya092 — APPROVED** [review]

**Review by Sonya092 — DISMISSED** [review]

File: src/Components/sectionSearch/RotateLogo/RotateLogo.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+.circle {
+  position: absolute;
+  width: 96px;
+  height: 96px;
+  border-radius: 50%;
+}
+
+.circle1 {
+  background: url(../../../../public/assets/img/centerLogo.png);
```
Comment (Hacene010) [inline src/Components/sectionSearch/RotateLogo/RotateLogo.css]: You can do with better than ../../.. ;)

File: src/Components/sectionSearch/RotateLogo/RotateLogo.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  background-size: cover;
+  top: calc((50% - 48px) + 0px);
+  left: calc((50% - 48px) + 128px);
+}
+
+.circle2 {
+  background: url(../../../../public/assets/img/blue.png);
```
Comment (Hacene010) [inline src/Components/sectionSearch/RotateLogo/RotateLogo.css]: Same as the first comment

**Review by Hacene010 — CHANGES_REQUESTED** [review]
You can put your assets i the public folder and use /assets

File: src/Components/sectionSearch/RotateLogo/RotateLogo.css (hunk 2 above)
Comment (fortanza) [reply src/Components/sectionSearch/RotateLogo/RotateLogo.css]: I resolved problem of the photos links.

**Comment by LoicBrassart** [comment]
Mising SM's validation, please reassign me later

**Review by Sonya092 — APPROVED** [review]

**Review by Sonya092 — DISMISSED** [review]

**Review by Sonya092 — APPROVED** [review]

**Review by Hacene010 — APPROVED** [review]
that's better ;)

**Pull request merged**